lms/djangoapps/shoppingcart/views.py

```diff
@@ -494,9 +494,7 @@ def use_registration_code(course_reg, user):
     else:
         applicable_cart_items = [
             cart_item for cart_item in cart_items
-            if (
-                (isinstance(cart_item, PaidCourseRegistration) or isinstance(cart_item, CourseRegCodeItem))and cart_item.qty == 1
-            )
+            if isinstance(cart_item, (CourseRegCodeItem, PaidCourseRegistration)) and cart_item.qty == 1
```

Comment:
Note: This is a little different. The ‘and’ used to only apply to the second. Just making sure that was clear.

Reply:
There were parentheses around the `or` of the two `isinstance` checks, then the `and` applied after that.  This seems equivalent (and was suggested by pylint).

Reply:
Misread the parens then.  Great.  Thanks.